kubernetes/Makefile

```
@@ -0,0 +1,28 @@
+PORTNAME=	kubernetes
+DISTVERSIONPREFIX=v
+DISTVERSION=	1.30.2
+CATEGORIES=	devel
+
+MAINTAINER=	freebsd@sysctl.cz
+COMMENT=	Production-Grade Container Scheduling and Management.
```

```suggestion
COMMENT=	Production-grade container orchestration
```

Taken from <https://kubernetes.io/>.